src/main/java/io/weaviate/client/v1/backup/Backup.java

```diff
@@ -30,7 +27,11 @@ public BackupRestoreStatusGetter restoreStatusGetter() {
     return new BackupRestoreStatusGetter(httpClient, config);
   }
 
-//  public BackupGetter getter() {
-//    return new BackupGetter(config);
-//  }
+  public BackupCanceler canceler() {
+    return new BackupCanceler(httpClient, config);
+  }
+  
+  //  public BackupGetter getter() {
+  //    return new BackupGetter(config);
+  //  }
```

Comment:
can we remove this commented code?

Reply:
I removed this and the related commented-out test too (it's quite simple to recreate later).